cosmwasm/its.ts

```diff
@@ -0,0 +1,172 @@
+import { CosmWasmClient } from '@cosmjs/cosmwasm-stargate';
+import { Command, Option } from 'commander';
+
+import { printError, printInfo } from '../common';
+import { ConfigManager } from '../common/config';
+import { ClientManager, mainProcessor, mainQueryProcessor } from './processor';
+
+export type TokenDataToRegister = {
+    tokenId: string;
+    originChain: string;
+    decimals: number;
+    supply?: string;
+    axelarId: string;
+};
+
+export async function registerToken(
+    interchainTokenServiceAddress: string,
+    client: ClientManager,
+    tokenDataToRegister: TokenDataToRegister,
+    dryRun: boolean,
+) {
+    const supply = tokenDataToRegister.supply;
+    const supplyParam = supply ? { tracked: String(supply) } : 'untracked';
+    const msg = {
+        register_p2p_token_instance: {
+            chain: tokenDataToRegister.axelarId,
+            token_id: formatTokenAddress(tokenDataToRegister.tokenId),
+            origin_chain: tokenDataToRegister.originChain,
+            decimals: tokenDataToRegister.decimals,
+            supply: supplyParam,
+        },
+    };
+
+    const [account] = await client.accounts;
+    printInfo('Registering token ', JSON.stringify(msg.register_p2p_token_instance));
```

Comment:
See how printInfo is defined, you don't need JSON conversion/spacing
```suggestion
    printInfo('Registering token', msg.register_p2p_token_instance);
```